ui/app.jsx

```diff
@@ -15,8 +15,9 @@ import DataManagement from 'pages/DataManagement/DataManagement'
 import Report from 'pages/Report/Report'
 import SummaryData from 'pages/SummaryData/SummaryData'
 import Login from 'pages/Login/components/Login'
-import ForgotPassword from 'pages/Login/components/ForgotPassword'
-import SetPassword from 'pages/Login/components/SetPassword'
+// 2021-05-07 mfranklin: Disable password reset
+// import ForgotPassword from 'pages/Login/components/ForgotPassword'
+// import SetPassword from 'pages/Login/components/SetPassword'
```

Comment:
Can we just delete these (and below)? We can always easily restore from git (and see the history / reasoning) later if we need to.

Reply:
Removed the old code :)